### ISSUE TITLE
Persisting settings?

### ISSUE THREAD

**Issue by quinnanya** [voyanttools/VoyantServer#21]
I might just not know where to look, but is there any way one could persist settings like custom stopword lists, color palettes, etc? I was hoping there might be some kind of folder with some kind of structured data for those things, but I haven't had any luck figuring out where that may be. Thank you!

**Comment by ajmacdonald** [comment]
Yes, it is possible to persist custom stopword lists and color palettes.

From the options dialog: after editing either of those you will get a new ID which can be used to retrieve the custom values.
You can then paste that ID into subsequent options dialogs (NB: this is currently broken for the palette but I am working on a fix).

Alternatively, you can append them to the URL, e.g. `corpus=austen&palette=903315baf27a070d6749041b2303bc43&stopList=keywords-205e93e6147d25aeefba1ddb3c011962`.
(If you export an URL for your corpus it will automatically include these custom parameters in the URL.)

![image](https://user-images.githubusercontent.com/882444/171217894-af17398e-b7ee-4daa-8fa5-57822020b2a9.png)

This is standard Voyant functionality. Currently there is no support for adding your own presets to a folder for use with VoyantServer.
